app/models/assignment.rb

```diff
@@ -1,4 +1,9 @@
 class Assignment < ApplicationRecord
   belongs_to :client
   belongs_to :volunteer
+  belongs_to :user
```

Comment:
One option to make this clearer is to call the association `creator` instead, i.e. something like `belongs_to :creator, class_name: User` (= column name `creator_id`)